theme/styles/snippets/news-feed.scss

```diff
@@ -0,0 +1,77 @@
+.NewsFeed {
+  margin-left: $site-padding-x-mobile;
+
+  @include media('xs-up') {
+    margin-right: $site-padding-x-mobile;
+  }
+
+  @include media('md-up') {
+    margin-left: $site-padding-x-desktop;
+    margin-right: $site-padding-x-desktop;
+  }
+
+  &__table-container {
+    overflow-x: scroll;
+    width: 100%;
+
+    table {
+      overflow-x: scroll;
+      overflow: hidden;
+      width: 100%;
+    }
+  
+    tr {
+      display: flex;
+      align-items: center;
+      min-width: 45rem;
+      padding-right: $site-padding-x-mobile;
+      border-bottom: 1px solid color('black');
+      padding: 1.25rem 0.75rem 1.25rem 0;
+      
+      a:hover {
+        @extend %transition-short;
+        color: lighten(black, 25%);
+      }
+
+      @include media('xs-up') {
+        padding: 0.75rem 0 0.75rem 0;
+      }
+
+      @include media('lg-up'){
+        min-width: 55rem;
+      }
+  
+      > :first-child {
+        width: 10%;
+  
+        img {
+          width: 3.4375rem;
+          height: 3.4375rem;
+      
+          @include media('lg-up') {
+            width: 4rem;
+            height: 4rem;
+          }
+        }
+      }
+  
+      > :nth-child(2)  {
+        @extend .serif-lg;
+        width: 70%;
+        
+        @include media('lg-up') {
+          min-width: 50rem;
+          padding: 0 1.5rem;
+        }
+      }
+  
+      > :nth-child(3)  {
+        width: 20%;
```

Comment:
This works but I'm not sure if `first-child`, `nth-child(2)`, `nth-child(3)` is a good way to select these! (I'm selecting the first child (1st col), second child (2nd col), third child (3rd col) of the table)